Rakefile

```diff
@@ -4,9 +4,7 @@ require "rubocop/rake_task"
 
 ENV["RAILS_ENV"] ||= "test"
 
-RSpec::Core::RakeTask.new(:spec).tap do |rspec_task|
-  rspec_task.rspec_opts = "--fail-fast"
-end
+RSpec::Core::RakeTask.new(:spec)
```

Comment:
For that I would rather add a line like this:

```Ruby
rspec_task.rspec_opts += " --seed #{ENV["SEED"]}" if ENV["SEED"]
```

right after the fail fast one, so you can then run the tests with the same seed by just doing:

```Shell
SEED=1212 rake
```
and then, we still get fast failures when running on travis, but one can still run the tests in a deterministic order... what do you think?